src/App.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './types/Tab';
```

Comment:
Please add all files to your commit to fix this problem
![image](https://user-images.githubusercontent.com/96409563/197722595-1c9994c0-7449-4a08-84e0-1a58fd651e26.png)